axelor-stock/src/main/resources/views/StockMove.xml

```diff
@@ -881,6 +882,10 @@
     <call class="com.axelor.apps.stock.web.StockMoveController" method="updateStocks"/>
   </action-method>
 
+	<action-method name="action-stock-method-show-sale-order-lines">
```

Comment:
action name should be 'action-stock-**move**-method-**view**-sale-order-lines'.

Also, change it from action-method to action-view.